paddle/framework/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
```

Comment:
I am only enforcing Google C++ style for new directories.  Do you mean we call cpplint to check new code?  I'd love to have it, but it seems cannot reformat the code. Do we really need it?